src/main/java/dev/sapphic/wearablebackpacks/Backpack.java

```diff
@@ -10,131 +10,131 @@
 import net.minecraft.util.collection.DefaultedList;
 import net.minecraft.util.math.BlockPos;
 import net.minecraft.world.BlockView;
-import org.checkerframework.checker.nullness.qual.Nullable;
+import org.jetbrains.annotations.Nullable;
 
 public interface Backpack {
-  String ROWS = "Rows";
-  String COLUMNS = "Columns";
+    String ROWS = "Rows";
```

Comment:
Changing the indentation of every file is out of scope for a version port.

Reply:
It's a habit of mine